security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisResultBuilder.java

```diff
@@ -26,8 +28,8 @@
 public class SecurityAnalysisResultBuilder {
 
     private final LimitViolationFilter filter;
-    private final RunningContext context;
-    private final List<SecurityAnalysisInterceptor> interceptors;
+    protected final RunningContext context;
```

Comment:
Since we should not anymore subclass the builder, is it necessary / useful to make it protected?